core/src/components/datetime/test/minmax/datetime.e2e.ts

```diff
@@ -105,4 +108,15 @@ test.describe('datetime: minmax', () => {
       expect(await minutes.count()).toBe(60);
     });
   });
+
+  test('setting value outside bounds should show in-bounds month', async ({ page }) => {
```

Comment:
Can we add tests for when only the `min` + `value` is defined and another for when only the `max` + `value` is defined?